### DISCUSSION TITLE
Weblate translation

### DISCUSSION THREAD

**Discussion by Mis1ify** [jeffvli/feishin#342] [General]
Hi, could you add support for translating to Polish language in Weblate? I would like to help with the project.

**Comment by jeffvli** [comment]
You can add languages by yourself by clicking the plus button on the [Translation](https://hosted.weblate.org/projects/feishin/translation/) page.

![image](https://github.com/jeffvli/feishin/assets/42182408/deb16f58-da14-475d-a4de-d33a3501a5bb)

**Comment by Mis1ify** [comment]
Oh, okay, I see now! Thanks!
